### PR TITLE
Increase learning rate and make it easier to plot batches

### PR DESCRIPTION
**Pull request by Katsutoshii**
Increase learning rate and make it easier to plot batches.
Also include PHX data in training batch.

### CODE REVIEW

**Comment by Katsutoshii** [comment]
MSE @1000 epochs = 0.0037:
image

**Review by SallyElHajjar — APPROVED** [review]
